File: Data Structures & Algorithms/remove-element/submission-2.cpp
class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
        int write = 0;
        for (int i = 0; i < nums.size(); i ++) {
            int elem = nums[i];
            if (elem == val) {
                continue;
            }
            nums[write++] = elem;
        }
        return write;
    }
};

File: Data Structures & Algorithms/two-integer-sum/submission-2.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        hashMap = {}

        for index, elem in enumerate(nums):
            if target - elem in hashMap:
                return [hashMap[target - elem], index]
            hashMap[elem] = index

        return []